test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js

```diff
@@ -115,36 +115,36 @@ describe('ExportMfpToolContent', () => {
 			spyOn(mfpServiceMock, 'getCapabilities').and.returnValue(capabilities);
 			const element = await setup({ ...mfpDefaultState, current: initialCurrent });
 
-			expect(element.shadowRoot.querySelectorAll('#select_layout option')).toHaveSize(2);
+			expect(element.shadowRoot.querySelectorAll('.layout-button')).toHaveSize(2);
 			expect(element.shadowRoot.querySelectorAll('#select_scale option')).toHaveSize(3);
```

Comment:
Would expect here also a test for the presence of the active css class on the correct position